lib/private/Avatar/AvatarManager.php

```diff
@@ -65,6 +68,15 @@ class AvatarManager implements IAvatarManager {
 	/** @var IConfig */
 	private $config;
 
+	/** @var IServerContainer */
+	private $serverContainer;
+
+	/** @var Coordinator */
+	private $bootstrapCoordinator;
+
+	/** @var IAvatarProvider[] */
+	private $providers;
```

Comment:
```suggestion
	private $providers = [];
```

can be used instead of the initialization in the constructor